sdk/python/tests/conftest.py

```diff
@@ -24,13 +25,15 @@
 import pytest
 from _pytest.nodes import Item
 
-from feast import FeatureStore
-from feast.wait import wait_retry_backoff
-from tests.data.data_creator import create_basic_driver_dataset
-from tests.integration.feature_repos.integration_test_repo_config import (
+os.environ["FEAST_USAGE"] = "False"
+os.environ["IS_TEST"] = "True"
```

Comment:
I think we should wrap this wrap these somehow so that we set them when running tests and restore the old values once we're done running tests. Pytest should have a way of making this possible